content/reference/endpoints/reviews.md

```diff
@@ -3,44 +3,134 @@
-rating           | Integer | Read       | Review's rating.
-title            | String  | Read       | Review's title.
------------------|---------|------------|------------
-created_at       | [Time](/reference/enums#formats) | Read       | Review's create time.
-updated_at       | [Time](/reference/enums#formats) | Read       | Review's update time.
-{: class="table table-bordered"}
-  </div>
-  <div class="tab-pane" id="reviews_write" markdown="1">
-Name             | Type    | Read/Write | Description
------------------|---------|------------|------------
-id               | Integer | Read       | Review's id.
------------------|---------|------------|------------
-comment          | Text    | Read/Write | **Required**. Review's comment. (10 min, 2000 max characters)
-country_code     | String  | Read       | Reviewer's country code. e.g: FR
-name             | String  | Read       | Reviewer's name, cropped to show only the initial of the last name.
-rating           | Integer | Read/Write | **Required**. Review's rating. (between 1 and 6)
-title            | String  | Read/Write | Review's title. (3 min, 100 max characters)
------------------|---------|------------|------------
-created_at       | [Time](/reference/enums#formats) | Read       | Review's create time.
-updated_at       | [Time](/reference/enums#formats) | Read       | Review's update time.
-{: class="table table-bordered"}
-  </div>
-</div>
+   <div class="tab-pane active" id="public" markdown="1">
+ Group    | Subgroup            | Name                  | Type    | Read/Write | Description
+ ---------|---------------------|-----------------------|---------|------------|------------
+          |                     | account               | Integer | Read       | Account's ID related to the Review
+          |                     | rental                | Integer | Read       | Rental's ID related to the Review
+          |                     | source                | Integer | Read       | Source's ID related to the Review
+          |                     | reply                 | Integer | Read       | Reply's (ReviewReply)'s ID related to the Review
+          |                     | booking               | Integer | Read       | Booking's ID related to the Review
+ ---------|---------------------|-----------------------|---------|------------|------------
+          |                     | id                    | Integer | Read       | Review's ID. 
+          |                     | title                 | Text    | Read | Title/Summary of the review. 
+          |                     | comment               | Text    | Read | Comment from traveler (guest). 
+          |                     | private_comment       | Text    | Read | Private comment from the traveler (guest) (not meant to be exposed publicly). Requires `reviews_read` or `reviews_write` scope 
+          |                     | rating                | Integer | Read/Write | Overall rating of a stay 
```

Comment:
as far as I recall this was changed to a decimal already:
`required(:rating).filled(:decimal, gteq?: 1, lteq?: 5)`

Reply:
Are you sure? 
The idea was the have rolling migration, as far as I recall, we accept decimal on write, but for read this value is passed directly as `rating_accurate` and casted to integer for `rating` (for backwards compatibility)

Reply:
Yeah, exactly, it's a decimal on write and integer on read and there is a rating accurate that is a decimal

Reply:
hmm, didn't know that, thanks for clarification and heads up Mat 👍